api/app_analytics/views.py

```diff
@@ -73,8 +73,20 @@ def post(self, request, *args, **kwargs):
                 content_type="application/json",
                 status=status.HTTP_200_OK,
             )
+
+        # Since the post body data has bare names of features with counts,
+        # if we put the identifier in the post body it could collide with
+        # a feature name. It's more straightforward to set it as a query parm.
+        identifier = request.query_params.get("identifier")
+        if identifier:
+            assert isinstance(identifier, str)
+        else:
+            identifier = None
```

Comment:
Let's add a TODO to fix this in v2 as this is ugly. We can also an item to the issue here: https://github.com/Flagsmith/flagsmith/issues/1667

Reply:
Ok sounds good. I've added the TODO locally and I added a comment to #1667 about it as well.